esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1125,7 +1125,7 @@ unsigned WsWuInfo::getWorkunitThorLogInfo(IArrayOf<IEspECLHelpFile>& helpers, IE
                 thorLog->setProcessName(processName.str());
                 thorLog->setClusterGroup(groupName.str());
                 thorLog->setLogDate(logDate.str());
-                thorLog->setNumberSlaves(numberOfSlaves);
+                thorLog->setNumberSlaves(numberOfSlaveLogs);
```

Comment:
Probably should rename this to thorLog->setNumberSlaveLogs() but does that require many other changes in other files ?